charts/sonarqube-dce/Chart.yaml

```diff
@@ -24,6 +24,8 @@ maintainers:
     email: jeremy.cotineau@sonarsource.com
 annotations:
   artifacthub.io/changes: |
+    - kind: fixed
+      description: "Fixed unsupported wget parameter `--proxy off` with `--no-proxy`"
```

Comment:
are these descriptions rendered as markdown by the changelog section of ArtifactHub.io?